git-tar/function/ops.go

```diff
@@ -487,3 +485,18 @@ func getShortSHA(sha string) string {
 	}
 	return sha[:7]
 }
+
+func formatTemplateRepos() []string {
+	templateRepos := []string{"https://github.com/openfaas/templates",
+		"https://github.com/openfaas-incubator/node8-express-template.git",
+		"https://github.com/openfaas-incubator/golang-http-template.git"}
+
+	if envTemplates := os.Getenv("custom_templates"); len(envTemplates) > 0 && strings.Contains(envTemplates, "https://") {
+		customTemplates := strings.Split(envTemplates, ",")
```

Comment:
This code should really validate each URL. I'll merge, but we need to fix up after that, also https should not be a requirement for URLs.